### ISSUE TITLE
Dataview Query (optional) doesn't work

### ISSUE THREAD

**Issue by DTIM001** [mdelobelle/metadatamenu#165]
# Dataview Query (optional) doesn't work
- when I add the Dataview query for the field Area, it doesn't work.
![Snipaste_2022-11-06_22-00-41](https://user-images.githubusercontent.com/113022813/200175072-a1c1c5f7-cbf9-42aa-a58b-9a56c7285634.png)
![Snipaste_2022-11-06_21-59-44](https://user-images.githubusercontent.com/113022813/200175082-58db4adf-e452-4650-9a6d-06b27cb1823e.png)

**Comment by mdelobelle** [comment]
Hi, does your note has a fileClass defined in its frontmatter?
Do you see any error in the console?

**Comment by toonbear** [comment]
Same issue
I try to change obsidian language to english but still not work
My windows system language is chinese

**Comment by BarryBurton57** [comment]
Same issue also. 

I can see an error in the console:
```
Uncaught (in promise) TypeError: o.toLowerCase is not a function
    at eval (plugin:metadata-menu:2099:72)
```

**Comment by mdelobelle** [comment]
should be fixed in 0.6.1
